assembly/build.gradle

```diff
@@ -97,6 +100,19 @@ shadowJar {
 
 
         include(dependency("org.eclipse.jetty:.*"))
+        include(dependency("org.eclipse.jetty.orbit:.*"))
```

Comment:
Thank you for the fixes!

Reply:
What do you think about switching to format which we discussed (and used in benchmarks)? ->  transitively include everything and exclude what we don't need as it is hard to track all the dependencies

Reply:
Yes, I will do it in a separate PR.